AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -818,7 +820,10 @@ protected Void doInBackground(Void... params) {
               Constants.loggingEnabled = true;
             }
 
-            parseSdkCounters(response);
+            HashSet<String> enabledCounters = parseSdkCounters(response);
```

Comment:
tip:
When defining types, use `Set<x>` instead of the concrete class `HashSet<x>`. Even though it doesn't matter a lot in this particular instance, you build the habit of retaining as much flexibility as possible. By defining it as `Set` you retain the flexibility of how exactly to implement the set. However, if you define it as a `HashSet`, then if you find a more optimized solution in the future, you can't simply change the backing implementation